puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foremanRelease.groovy

```diff
@@ -35,7 +35,12 @@ pipeline {
                 withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'jenkins-centos', passwordVariable: 'PASSWORD', usernameVariable: 'USERNAME']]) {
                     runPlaybook(
                         playbook: 'ci/centos.org/ansible/jenkins_job.yml',
-                        extraVars: ["jenkins_job_name=foreman-nightly-test", "jenkins_username=${env.USERNAME}", "jenkins_password=${env.PASSWORD}"],
+                        extraVars: [
+                            "jenkins_job_name=foreman-nightly-test",
+                            "jenkins_username=${env.USERNAME}",
+                            "jenkins_password=${env.PASSWORD}"
+                            "jenkins_job_link_file=${env.WORKSPACE}jobs/foreman-nightly-test"
```

Comment:
Does `env.WORKSPACE` include a trailing slash? Note this is repeated a few times in the PR

Reply:
Good catch, I missed that when updating from test job.